plugins/cilium-cni/cilium-cni.go

```diff
@@ -630,6 +692,13 @@ func cmdDel(args *skel.CmdArgs) error {
 		log.WithError(err).Warning("Errors encountered while deleting endpoint")
 	}
 
+	// TODO: following the example of the bridge reference plugin here.
+	// If Netns is empty, call the delegated IPAM plugin DEL to release any IPs,
+	// but don't need to do anything else afterwards.
+	if conf.IpamMode == ipamOption.IPAMDelegatedPlugin && args.Netns == "" {
```

Comment:
I'm not sure I quite understand this case. When would `args.Netns` be empty?

Reply:
I cargo-culted this from the bridge reference plugin. Will check the commit history to see if there's any more context about why they added this.